File: IOS/Claude45/appomodoro/appomodoro/Models/PomodoroPhase.swift
//
//  PomodoroPhase.swift
//  appomodoro
//
//  Created by JOSE ZARABANDA on 10/8/25.
//

import Foundation

/// Represents the different phases of a Pomodoro session
enum PomodoroPhase: String, Codable, CaseIterable {
    case focus = "Focus"
    case shortBreak = "Short Break"
    case longBreak = "Long Break"
    
    var systemImageName: String {
        switch self {
        case .focus:
            return "brain.head.profile"
        case .shortBreak:
            return "cup.and.saucer.fill"
        case .longBreak:
            return "bed.double.fill"
        }
    }
    
    var description: String {
        self.rawValue
    }
}

File: Android/Claude_45/app/src/main/java/com/zarabandajose/appomodoro/model/PomodoroPhase.kt
package com.zarabandajose.appomodoro.model

enum class PomodoroPhase(val displayName: String, val defaultDurationSeconds: Int) {
    FOCUS("Focus", 25 * 60),
    SHORT_BREAK("Short Break", 5 * 60),
    LONG_BREAK("Long Break", 15 * 60),
    CUSTOM("Custom", 10 * 60)
}
